applications/coreutils/chmod.cpp

```diff
@@ -0,0 +1,34 @@
+#include <libsystem/io/Filesystem.h>
+#include <libsystem/io/Stream.h>
+#include <libsystem/utils/NumberParser.h>
+
+int main(int argc, char **argv)
+{
+    if (argc == 1)
```

Comment:
You should also add if you have only 2 arguments (for example `chmod 750` and no file name after that)